<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caesar_Cyper</class>
 <widget class="QWidget" name="Caesar_Cyper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caesar cyper</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>caesar.ico</normaloff>caesar.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #4facfe, stop:1 #00f2fe);</string>
  </property>
  <widget class="QLineEdit" name="input">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>501</width>
     <height>91</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 200); /* Легкая прозрачность */
    border-radius: 8px; /* Закругленные углы */
    border: 1px solid #b1b1b1; /* Граница вокруг текстового поля */
    padding: 5px; /* Немного отступа внутри поля */
    color: #333; /* Цвет текста */</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_key">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 200); /* Легкая прозрачность */
    border-radius: 8px; /* Закругленные углы */
    border: 1px solid #b1b1b1; /* Граница вокруг текстового поля */
    padding: 5px; /* Немного отступа внутри поля */
    color: #333; /* Цвет текста */</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent; /* Прозрачный фон */
    color: #555; /* Цвет текста */
    font-size: 36px; /* Размер шрифта */
    font-weight: bold; /* Жирное начертание */
}</string>
   </property>
   <property name="text">
    <string>Caesar cypher</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent; /* Прозрачный фон */
    color: #555; /* Цвет текста */
    font-size: 16px; /* Размер шрифта */
    font-weight: bold; /* Жирное начертание */
}</string>
   </property>
   <property name="text">
    <string>Write phrase/cypher:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent; /* Прозрачный фон */
    color: #555; /* Цвет текста */
    font-size: 16px; /* Размер шрифта */
    font-weight: bold; /* Жирное начертание */
}</string>
   </property>
   <property name="text">
    <string>Write key:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encryptButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #57c9f2;
    border-radius: 5px; /* Закругленные углы для кнопки */
    border: 2px solid #47b2e1; /* Граница вокруг кнопки */
    color: #555; /* Цвет текста на кнопке */
    padding: 5px 10px; /* Внутренние отступы */
    font-size: 16px; /* Размер шрифта текста кнопки */
}

QPushButton:hover {
    background-color: #47b2e1; /* Цвет кнопки при наведении */
    color: white; /* Изменение цвета текста при наведении */
}

QPushButton:pressed {
    background-color: #3a9dbd; /* Цвет кнопки при нажатии */
    color: white; /* Изменение цвета текста при нажатии */
}
</string>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decryptButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #57c9f2;
    border-radius: 5px; /* Закругленные углы для кнопки */
    border: 2px solid #47b2e1; /* Граница вокруг кнопки */
    color: #555; /* Цвет текста на кнопке */
    padding: 5px 10px; /* Внутренние отступы */
    font-size: 16px; /* Размер шрифта текста кнопки */
}

QPushButton:hover {
    background-color: #47b2e1; /* Цвет кнопки при наведении */
    color: white; /* Изменение цвета текста при наведении */
}

QPushButton:pressed {
    background-color: #3a9dbd; /* Цвет кнопки при нажатии */
    color: white; /* Изменение цвета текста при нажатии */
}
</string>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="output">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 200); /* Легкая прозрачность */
    border-radius: 8px; /* Закругленные углы */
    border: 1px solid #b1b1b1; /* Граница вокруг текстового поля */
    padding: 5px; /* Немного отступа внутри поля */
    color: #333; /* Цвет текста */</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>730</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);
border: none;
font-size: 16px;
color: #555;
font-weight: bold; 
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/backButton.png</normaloff>../img/backButton.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent; /* Прозрачный фон */
    color: #555; /* Цвет текста */
    font-size: 16px; /* Размер шрифта */
    font-weight: bold; /* Жирное начертание */
}</string>
   </property>
   <property name="text">
    <string>Result:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
